### ISSUE TITLE
Question on proper working scenario of Copy Last Frame 

### ISSUE THREAD

**Issue by jeenuine** [Microsoft/VoTT#259]
I've tried tagging on video with the enabled option of Copy Last Frame. I expected that bounding boxes on the first frame show in the same position on the next frame. However, the result is not what I've thought. the result is like the pictures below. If you compare both of pictures below, you can find the bounding boxes are moved to the right side. Is this proper result with the option "Copy Last Frame" on? I used the latest version 1.5.0 on Windows 10 and Ubuntu 18.04. The output is the same in both environments. 

First Frame
![image](https://user-images.githubusercontent.com/9638641/47125541-2409ab00-d2bf-11e8-8325-e48e8b677bfe.png)

The next Frame
![image](https://user-images.githubusercontent.com/9638641/47125597-5915fd80-d2bf-11e8-8b8f-8a4c108015a1.png)

If this is not the right behavior of the VoTT. Is there any way I can solve this out?

**Comment by aribornstein** [comment]
Make sure your pictures are the same dimensions positioning in copy next frame is relative not absolute.

**Comment by gnoopy** [comment]
Hi @aribornstein. 
I have the same experience. But there's something I misunderstand on the feature "copy next frame" of VoTT. Could you elaborate on "the same dimensions positioning"?

It looks like both of pictures keep the marked products in the same position and also the shots show absolutely same space with the same dimension. The bounding boxes in the first frame moved to the right of the second frame.

Is there any algorithm of "copy next frame" we have to know to use it with the proper way?

**Comment by MartinBrisiak** [comment]
i had the same issue, last working version for me was 1.0.9, i didn't see working "copy last frame" feature since.

**Comment by aznedy** [comment]
I agree with MartinBrisiak, the last version which had this feature working is 1.0.9. I have tried every release version since, and none of them work. 

Another user suggested a workaround in a previous issue that I opened, which is to open the debug console, which makes the tag box appear on the frame. Closing the debug console also has the effect. Unfortunately opening or closing the debug console after tagging every video frame is really not a tenable solution for my team. So for the time being we're staying on version 1.0.9. I really want to move to a more recent release, but without this feature working, we're stuck on this old build.
